docs/content/docs/reference/nextjs/configuration.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: 'Configuration'
+---
+
+## Configuration
+
+To configure Nhost with Next.js follows the same logic as with React, except we are intialising with `NhostSSR` instead of `Nhost`.
+Under the hood, `NhostSSR` uses cookies to store the refresh token, and disables auto-refresh and auto-login when running on the server side.
```

Comment:
"Configuring Nhost with Next.js follows the same logic as React, except we are initializing with `NhostSSR` instead of `Nhost`.
Under the hood, `NhostSSR` uses cookies to store the refresh token, and disables auto-refresh and auto-login when running on the server-side."